src/gui/main_window.cpp

```diff
@@ -67,6 +67,8 @@ constexpr int MainWindow::max_recent_files;
 
 int MainWindow::num_open_files = 0;
 
+int const MainWindow::EXIT_CODE_REBOOT = -123456789;
```

Comment:
I would prefer a value more close to 0. And this CAPITAL_STYLE looks very much like an preprocessor macro.